src/cloud-code/Parse.Cloud.js

```diff
@@ -37,6 +37,7 @@ var ParseCloud = {};
  * @memberof Parse.Cloud
  * @param {String} name The name of the Cloud Function
  * @param {Function} data The Cloud Function to register. This function can be an async function and should take one parameter a {@link Parse.Cloud.FunctionRequest}.
+ * @param {Function} validator An optional function to help validating cloud code. This function can be an async function and should take one parameter a {@link Parse.Cloud.FunctionRequest}. Returning false from this function will return a "validation fail" error.
```

Comment:
Change the @links to the proper request type. They are all FunctionRequest

Reply:
I will add the cloud code docs around the validator object / function once the everything else is sorted so I have a clear understanding on what the validator will look like, if that's ok?

Reply:
This is for the [API Reference](https://parseplatform.org/parse-server/api/4.3.0/Parse.Cloud.FunctionRequest.html). You can `npm run docs` to generate and test it.

Reply:
Yeah, that's what I meant sorry. Once I have you're ready with the params of the validator, I'll update the API reference, just before this PR is possibly merged? I just don't want to write all the options for the validator that I have now into the API, when others may have different opinions of how keys should be named, etc.

Unless you're happy with the way it's laid out now, then i'm happy to work on the API reference.

Reply:
We can wait, just leave this conversation open.